pkg/graphite.go

```diff
@@ -150,7 +149,7 @@ func (m *Metrics) fetchBPFMetrics() (map[string]bpfMetrics, error) {
 	}
 	defer func() {
 		if err := file.Close(); err != nil {
-			log.Fatal(err)
```

Comment:
Should probably read `log.Error`?

Reply:
you don't believe it, but standart golang log library doesn't know anything about error logging level:
https://golang.org/pkg/log/